articles/configuring-maintenance-windows-in-fleet.md

```diff
@@ -0,0 +1,110 @@
+
+1. On your Fleet UI, go to the **Policies** tab.
+2. On the top left dropdown, select a team if needed.
+3. On the top right, click **Manage automations** → **Calendar events**.
+4. On the top left corner of the newly opened modal, click **Enable**.
+5. Paste the webhook URL you have in the box named: **Resolution webhook URL**
+6. Select one or more desired policies (click the checkbox next to it)
+
+Video: [Setting a webhook for failing policies in Fleet](https://www.youtube.com/watch?v=evMB6zC8jso). (3m)
+
+### Which users will have an event set for them?
+
+If a policy on a host belonging to a Google Calendar user fails, an event will be set on their calendar next Tuesday between 9 a.m. and 5 p.m. local time.
+
+If more than one host has the same user, the event will only be set for one of the hosts. Once this host has been remedied, another event will be scheduled for the next of the user’s hosts, and so forth.
+
+If more than one user is set for a specific host, the event will be set for one of the users.
+
+### What details will be presented in the calendar event?
+
+When adding a policy (**Policies** tab > Top right corner **Add policy** > **create the policy** > Press **Save**), a modal will open with the policy details like this:
+
+![Save policy dialog with Antivirus healthy for Linux with example AI-generated description and resolution.](../website/assets/images/articles/configuring-maintenance-windows-in-fleet-651x782@2x.png "Save policy dialog")
+
+The description and resolution fields can be AI-generated based on the SQL on the policy.
+
+The admin may press **Autofill** one time to regenerate or edit the text manually. Learn more about [Fleet’s AI-assisted policy descriptions and resolutions](https://fleetdm.com/guides/fleet-ai-assisted-policy-descriptions-and-resolutions).
+
+These fields will appear in the details of the scheduled event.
+
+### Rescheduling the maintenance window (Host owner/user)
+
+* **Rescheduling a maintenance window**: Users can reschedule their maintenance window by moving it on their calendar. Fleet will respect the new schedule and will run the maintenance webhook at the new time.
+* **Rescheduled to the past**: If the user reschedules the event to a time before the current time, the Fleet server will reschedule it to the day after the original date.
+* **Deleted maintenance window**: If the user deletes an event, the Fleet server reschedules it to the day after the original date. If it is deleted again, it will be rescheduled for the following day.
+
+> The Fleet server handles monitoring the user’s calendar event in real time. Changes to the scheduled event will typically be monitored and addressed within 30 seconds. However, to save load on the Google calendar, only the next week is monitored. In the case of the event being rescheduled to more than a week from the present time, it will still be addressed within 30 minutes rather than in real-time.
```

Comment:
```suggestion
> The Fleet server handles monitoring the user’s calendar event in real time. For example, if the end user deletes the calendar event it will be rescheduled within 30 seconds. However, to save load on the Google calendar, only the next week is monitored. In the case of the event being rescheduled to more than a week from the present time, it will still be addressed within 30 minutes rather than in real-time.
```

I think providing the "end user deletes calendar" example here would be helpful.